brax/jumpy.py

```diff
@@ -30,17 +33,21 @@
 float32 = onp.float32
 int32 = onp.int32
 
+FORCE_JNP = os.getenv("JUMPY_FORCE_JNP", 'False').lower() in ('true', '1', 't')
 
 def _in_jit() -> bool:
   """Returns true if currently inside a jax.jit call."""
+  if FORCE_JNP:
+    return True
   return core.cur_sublevel().level > 0
 
 
 def _which_np(*args):
-  """Returns np or jnp depending on args."""
-  for a in args:
-    if isinstance(a, jnp.ndarray) and not isinstance(a, onp.ndarray):
-      return jnp
+  checker = lambda a: (
```

Comment:
interesting - would you want to support FORCE_JNP here too?  or not?

Reply:
This is a separate issue, but I think it would be better if brax always used `onp` explicitly in cases where `jnp` would not work. With that change, you could make it possible to choose the jumpy backend.